### PR TITLE
w01d01

### PR DESCRIPTION
**Pull request by jorgeRafaelSJ**
comfort:5
completion:5

All was good!

### CODE REVIEW

File: jorgeRafaelSJ/day1.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+	</head>
+	<body>
+		<h1>HELLO! That was a great first day warm up into GA life!</h1>
+
+
+		<ol>
+			<li>Today I got a clearer understanding of GitHub and the purposes of fork, clone, add, commit, push, or pull requests. It had been a little harder the first time around on the pre-course work.</li>
+
+			<li class = "question">I understand how to link a separate CSS file to the header section. What is the standard way of organizing related files on Git and pushing them onto the remote git in an orderly fashion?</li>
```
Comment (tgaff) [inline jorgeRafaelSJ/day1.html]: we'll get to this; short answer "it depends"

File: jorgeRafaelSJ/passion_project.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+			h1, h2 {
+				border-bottom: 1px solid black;
+			}
+
+		</style>
+	</head>
+	<body>
+		<h1>Passion Project</h1>
+
+		<p>I have been thinking for some time about figuring out a way to make classical music more accessible for curious new listeners. Most people are turned off by the "pretentiousness" of the art environment because of how obscure it can seem. I would love to come up with a solution that can show a curious listener that:</p>
```
Comment (tgaff) [inline jorgeRafaelSJ/passion_project.html]: interesting; are you into classical music?

**Comment by tgaff** [comment]
Great work~! :cactus:

File: jorgeRafaelSJ/passion_project.html (hunk 2 above)
Comment (jorgeRafaelSJ) [inline jorgeRafaelSJ/passion_project.html]: Hi Travis! Just seeing your comments now. Yes, I am into classical, a lot. I did my undergrad in Piano Performance. I would love to work on something music related. :)

**Pull request merged**